### PR TITLE
created annotation and validator class for dtos

### PR DESCRIPTION
**Pull request by SevoMkrtchyan**

### CODE REVIEW

File: src/validator/DtoValidator.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+public class DtoValidator {
+    private static final String REGEX = "^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$";
+
+    public List<String> validateDto(Object dto) throws IllegalAccessException {
+        List<String> messages = new LinkedList<>();
+        Field[] declaredFields = dto.getClass().getDeclaredFields();
+        for (Field declaredField : declaredFields) {
+            declaredField.setAccessible(true);
+            if (declaredField.isAnnotationPresent(Length.class)) {
```
Comment (LevonAloyan) [inline src/validator/DtoValidator.java:21]: Please change this method and split it into separate private methods, your method is too big

Comment (LevonAloyan) [reply src/validator/DtoValidator.java:21]: Or even you can create separate validator classes for each annotation.

**Review by LevonAloyan — CHANGES_REQUESTED** [review]

File: src/validator/DtoValidator.java (hunk 1 above)
Comment (SevoMkrtchyan) [reply src/validator/DtoValidator.java:21]: Private methodnerov enem erevi, pahanji mej nshvac er 1 methodov

File: src/validator/DtoValidator.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+import java.time.LocalDate;
+import java.time.Period;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class DtoValidator {
+    private static final String REGEX = "^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$";
+
+    public static List<String> validateDto(Object dto) throws IllegalAccessException {
```
Comment (SevakMart) [inline src/validator/DtoValidator.java]: Instead of using Object, try to make this class Generic

Comment (SevoMkrtchyan) [reply src/validator/DtoValidator.java]: Done

**Pull request merged**